ec2utils/ec2utilsbase/Makefile

```diff
@@ -31,5 +31,13 @@ tar: clean
 test:
 	py.test tests/ec2utilsutilstest.py
 
+pypi:
```

Comment:
Sorry don't remember getting pinged about this review. Anyway, looks good. Although, it seems a bit redundant to build sdist in makefile when python setup.py already provides this functionality.

Reply:
sdist leaves stuff out, such as the man pages for example. If we put this on pypi then other packagers should get the complete source and not just what the Python tools consider important to Python.

Yes I tried playing around with additional settings in setup.py to include the man pages, but that was unreliable and I am not interested in changing the way the man pages are written just because the Python tools cannot cope properly.

Reply:
Yeah, it is confusing what flags do the trick given changes between distutils and setuptools. With IPA using [`include_package_data`](https://github.com/SUSE/ipa/blob/master/setup.py#L74) and recursive include of man in [MANIFEST](https://github.com/SUSE/ipa/blob/master/MANIFEST.in) seemed to do the trick.